helm/botkube/values.yaml

```diff
@@ -195,6 +195,9 @@ config:
   # about the best practices for the created resource
   recommendations: true
 
+  ssl:                                          # For using custom SSL certificates
```

Comment:
I think we should just comment out `ssl` section instead of setting the `ssl.cert` to a string which is "None". We shouldn't have done that before as well, can you please fix it?

Reply:
had to leave it uncommented due to [secret.yaml](https://github.com/infracloudio/botkube/blob/develop/helm/botkube/templates/secret.yaml) line 13
```yml
  ca-certificates.crt: {{ .Files.Get (printf "%s" .Values.config.communications.mattermost.cert) | b64enc }}
```
new line in pull is
```yml
  ca-certificates.crt: {{ .Files.Get (printf "%s" .Values.config.ssl.cert) | b64enc }}
```
which would cause errors if the ssl section was commented, without an extra if loop